server/numa-configuration.cpp

```diff
@@ -0,0 +1,104 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2022 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include <algorithm>
+#include <cassert>
+#include <sched.h>
```

Comment:
Already included in numa-configuration.h

Reply:
fixed